packages/block-library/src/nextpage/editor.scss

```diff
@@ -9,6 +9,7 @@
 
 	// Label
 	> span {
+		position: relative;
```

Comment:
```suggestion
```
This doesn't need to be added, it exists for this block already two lines below, just the more block was missing it

Reply:
>This doesn't need to be added

Ah, good eye! Thanks.